### PR TITLE
Add item list UI

### PR DESCRIPTION
**Pull request by Hawkey3s**

### CODE REVIEW

File: src/main/java/dekk/pw/pokemate/PokeMateUI.java [hunk 1]
```diff
@@ -164,6 +165,18 @@ public void mapInitialized() {
                             }
                         }
                         rows += "\"";
+                        String itemsList = "\"";
+                        for (Item item : context.getApi().getInventories().getItemBag().getItems()) {
+                            if(item.getCount() > 0) {
+                                String defaultImg = "icons/items/0.png";
+                                String imgSrc = "icons/items/" + item.getItemId().getNumber() + ".png";
+                                itemsList += "<td><img style=\'width: 50px; height: 50px; \' " +
```
Comment (SwipeX) [inline src/main/java/dekk/pw/pokemate/PokeMateUI.java:173]: Should also have a tr tag for each.

**Comment by SwipeX** [comment]
Can we have a screenshot of this? It looks like the container sizes have not been adjusted to add up to 12.

**Comment by Hawkey3s** [comment]
http://imgur.com/a/9WNIx

**Comment by BurakDev** [comment]
Nice feature ! +1 for merge

**Comment by SwipeX** [comment]
Ill need to edit the html, but we can use this.

**Comment by BurakDev** [comment]
@SwipeX What do you think about 2 buttons to hide/show items and pokemons ?

I can implement it when this PR is implemented.

**Comment by SwipeX** [comment]
There should be a pokemon and an item button on the navbar

**Pull request merged**